CMakeLists.txt

```diff
@@ -34,7 +34,8 @@ option(UAGENT_P2P_PROFILE "Build P2P discovery profile." ON)
 option(UAGENT_LOGGER_PROFILE "Build logger profile." ON)
 option(UAGENT_SECURITY_PROFILE "Build security profile." OFF)
 option(UAGENT_CLI_PROFILE "Build CLI profile." ON)
-option(UAGENT_BUILD_EXECUTABLE "Build MicroXRCE-DDS Agent provided executable." ON)
+option(UAGENT_BUILD_EXECUTABLE "Build Micro XRCE-DDS Agent provided executable." ON)
+option(UAGENT_BUILD_EXAMPLES "Build Micro XRCE-DDS Agent built-in usage examples" OFF)
```

Comment:
Should we rename this in order to be more explicit about "custom examples"?

Reply:
hmm, yes, we could keep the `UAGENT_BUILD_EXECUTABLE` and rename the other to `UAGENT_BUILD_USAGE_EXAMPLES`. What do you think?

Reply:
What is exactly the example? About how to use the library for creating a custom agent?

Reply:
Yes, that is correct. Anyway, we could use this flag to include further examples we want to provide the user with.